### ISSUE TITLE
released 1.5.1 gemfile is broken containing broken symlink

### ISSUE THREAD

**Issue by mtasaka** [ruby/syck#48]
Released 1.5.1 gemfile is broken. It contains broken symlink and installation fails:

```
$ gem install syck
ERROR:  While executing gem ... (Gem::Package::SymlinkError)
    installing symlink 'ext/syck/compile_commands.json' pointing to parent path /Users/ktsanaktsidis/Code/zendesk/syck/tmp/arm64-darwin23/syck/3.3.0/compile_commands.json of /home/tasaka1/.local/share/gem/ruby/gems/syck-1.5.1 is not allowed
	/usr/share/rubygems/rubygems/package.rb:433:in `block (2 levels) in extract_tar_gz'
	/usr/share/rubygems/rubygems/package/tar_reader.rb:65:in `each'
	/usr/share/rubygems/rubygems/package.rb:423:in `block in extract_tar_gz'
	/usr/share/rubygems/rubygems/package.rb:547:in `block in open_tar_gz'
	/usr/share/rubygems/rubygems/package.rb:544:in `wrap'
	/usr/share/rubygems/rubygems/package.rb:544:in `open_tar_gz'
	/usr/share/rubygems/rubygems/package.rb:422:in `extract_tar_gz'
	/usr/share/rubygems/rubygems/package.rb:396:in `block (2 levels) in extract_files'
	/usr/share/rubygems/rubygems/package/tar_reader.rb:65:in `each'
	/usr/share/rubygems/rubygems/package.rb:393:in `block in extract_files'
	/usr/share/rubygems/rubygems/package/file_source.rb:30:in `open'
	/usr/share/rubygems/rubygems/package/file_source.rb:30:in `with_read_io'
	/usr/share/rubygems/rubygems/package.rb:390:in `extract_files'
	/usr/share/rubygems/rubygems/installer.rb:861:in `extract_files'
	/usr/share/rubygems/rubygems/installer.rb:328:in `install'
	/usr/share/rubygems/rubygems/resolver/specification.rb:105:in `install'
	/usr/share/rubygems/rubygems/request_set.rb:195:in `block in install'
	/usr/share/rubygems/rubygems/request_set.rb:183:in `each'
	/usr/share/rubygems/rubygems/request_set.rb:183:in `install'
	/usr/share/rubygems/rubygems/commands/install_command.rb:207:in `install_gem'
	/usr/share/rubygems/rubygems/commands/install_command.rb:223:in `block in install_gems'
	/usr/share/rubygems/rubygems/commands/install_command.rb:216:in `each'
	/usr/share/rubygems/rubygems/commands/install_command.rb:216:in `install_gems'
	/usr/share/rubygems/rubygems/commands/install_command.rb:162:in `execute'
	/usr/share/rubygems/rubygems/command.rb:326:in `invoke_with_build_args'
	/usr/share/rubygems/rubygems/command_manager.rb:255:in `invoke_command'
	/usr/share/rubygems/rubygems/command_manager.rb:194:in `process_args'
	/usr/share/rubygems/rubygems/command_manager.rb:152:in `run'
	/usr/share/rubygems/rubygems/gem_runner.rb:56:in `run'
	/bin/gem:12:in `<main>'
```

**Comment by mtasaka** [comment]
Maintainer, would you release new version and re-package new gem with unneeded files cleaned up?

syck git repository does not contain `ext/syck/compile_commands.json` , but the released 1.5.1 gem contains this dangling symlink file.

**Comment by hsbt** [comment]
I will do that in the next week.

**Comment by hsbt** [comment]
I released v1.5.1.1. https://github.com/ruby/syck/releases/tag/v1.5.1.1
